modules/ipfs-cpinner-service/src/index.ts

```diff
@@ -45,8 +45,19 @@ const config: AuthConfig = {
 }
 
 const app = express()
+
 app.use(bodyParser.json())
-app.use(cors())
+
+app.use((req, res, next) => {
```

Comment:
it would be nice the have some comments here explaining why this was done as it was.

Reply:
```suggestion
// this enables cross-origin requests
app.use((req, res, next) => {
```